libr/core/cmd_anal.c

```diff
@@ -5268,38 +5276,36 @@ static int cmd_anal(void *data, const char *input) {
 			return false;
 		break;
 	case 'c':
-		if (input[1] == '?') {
-			eprintf ("Usage: ac [cycles]   # analyze instructions that fit in N cycles\n");
```

Comment:
Because it's already described in `a?`:
```nasm
[0x00000000]> a?~ac
| ac[?] [cycles]   analyze which op could be executed in [cycles]
| ai [addr]        address information (show perms, stack, heap, ...)
| at[?] [.]        analyze execution traces
[0x00000000]> 
```